packages/manager/src/views/DashboardPage/components/Analytics/Analytics.tsx

```diff
@@ -0,0 +1,235 @@
+import { Card, CardTitle, CardBody, CardHeader, Grid, GridItem } from '@patternfly/react-core';
+import { Chart, ChartAxis, ChartGroup, ChartLine, ChartThemeColor } from '@patternfly/react-charts';
+import { VictoryZoomContainer } from 'victory-zoom-container';
+import { TSPADeploymentCount } from '@app/services/analytics/types';
+import { UseQueryResult } from '@tanstack/react-query';
+
+const TotalDeploymentCardFields = ['Dev', 'QA', 'Stage', 'Prod'];
+
+type IGraphData = {
+  name: string;
+  x: string;
+  y: number;
+};
+
+type Props = {
+  QAData: IGraphData[] | undefined;
+  ProdData: IGraphData[] | undefined;
+  StageData: IGraphData[] | undefined;
+  DevData: IGraphData[] | undefined;
+  Totaldeployment: number | undefined;
+  TotalProperty: number;
+  averageTime: number | undefined;
+  lastMonthEphemeral: number | undefined;
+  TotalDeploymentData: UseQueryResult<TSPADeploymentCount[]>;
+};
+
+export const Analytics = ({
+  QAData,
+  ProdData,
+  StageData,
+  DevData,
+  Totaldeployment,
+  TotalProperty,
+  averageTime,
+  lastMonthEphemeral,
+  TotalDeploymentData
+}: Props) => {
+  let minCount = 0;
+  let maxCount = 0;
+  if (ProdData && StageData && QAData && DevData) {
```

Comment:
I would say move this to top and pass it as prop. U are already doing a sort and map. Then u can easily find the min and max values from there.